### PR TITLE
chore: Add headless Chrome parameters to work on ResinCi

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: patch
Depends-on: https://github.com/resin-io/resin-concourse/pull/248/
Signed-off-by: Thodoris Greasidis <thodoris@resin.io>

### CODE REVIEW

**Comment by thgreasi** [comment]
@resin-ci retest

**Comment by thgreasi** [comment]
@resin-ci retest

**Comment by thgreasi** [comment]
@resin-ci retest

File: karma.conf.coffee [hunk 1]
```diff
@@ -3,7 +3,17 @@ packageJSON = require('./package.json')
...
-	karmaConfig.browsers = ['ChromeHeadless']
+	karmaConfig.browsers = ['ChromeHeadlessCustom']
+	karmaConfig.customLaunchers =
+		ChromeHeadlessCustom:
+			base: 'ChromeHeadless'
+			flags: [
+				'--no-sandbox'
+				'--no-proxy-server'
+				'--disable-setuid-sandbox'
+				'--disable-web-security'
```
Comment (pimterry) [inline karma.conf.coffee]: Do we definitely need this one? This shouldn't be required to get the browser to run, it'll change the runtime behaviour and make it more lax (which probably isn't what we want for testing)

Comment (thgreasi) [reply karma.conf.coffee]: Already tried to keep the flags to the minimum but let my try removing this one as well now that the PR dust has settled.

Comment (thgreasi) [reply karma.conf.coffee]: It ends-up that for the final alpine node 10 image (which is used in the npm publish job) that we ended up we only need the `--no-sandbox` (at least for now).
I will revisit these later while trying to fix the concourse test for node 6 & node 8.

**Pull request merged**